src/test/kotlin/org/elm/ide/docs/ElmQuickDocumentationTest.kt

```diff
@@ -239,6 +239,29 @@ main = ()
 """,
             """
 <div class='definition'><pre><i>module</i> Main</pre></div>
+""")
+
+    // This test is kludgy: since a line comment before a doc comment will cause the doc comment to fail to attach to
+    // the module element, we need to put the line comment inside the doc comment.
+    fun `test module with docstring`() = doTest(
+            """
+module Main exposing (main)
+{-|  --^
+
+Module docs
+
+# Header
+@docs main
```

Comment:
When I was running your new code, I noticed that the module documentation's embedded links are broken when the list of identifiers in a section stretches across multiple lines. Here's an [example from Html.Events](https://github.com/elm/html/blob/master/src/Html/Events.elm#L20). 

<img width="430" alt="screen shot 2018-08-03 at 9 49 01 pm" src="https://user-images.githubusercontent.com/84525/43671397-36ca1ce0-9767-11e8-8b9e-ec5fd26988bb.png">